.bazelrc

```diff
@@ -84,14 +104,13 @@ try-import %workspace%/.bazelrc.user
 build:compdb --features=-layering_check
 
 # Compiling tests requires Java.
-common --java_runtime_version=remotejdk_11
+build --java_runtime_version=remotejdk_11
 
 # Coverage requires Java and GCC.
 coverage --config=coverage
 coverage --build_tests_only
 build:coverage --copt=-DNDEBUG
 build:coverage --combined_report=lcov
```

Comment:
How can we generate and upload this lcov coverage report? My understanding is that we need to build and link with special args to generate coverage, and that the bazel output is a `.dat` file which we can export to html (https://bazel.build/configure/coverage)

Reply:
Yeah, we can use `genthml` from the `lcov` package to generate the HTML version from the `.dat`.

Reply:
We would upload the original `lcov` format to the metrics gcs bucket, as `xxx_coverage.out`. The extension matters less as long as the file content is in `lcov` format. Can we comment about what tests are being included for coverage, and maybe a TODO if there is anything left out?

Reply:
Currently all cpp tests are included. They are run with various settings currently (XRT, PJRT), so I suppose I should investigate how to collect coverage for separate invocations.
I'll probably have to merge them, I can follow the current process and try to replicate it.

Reply:
I see that you only have python coverage for now, right?

Reply:
Yes, the pipeline is working.